src/styles/components/_c-landing.scss

```diff
@@ -20,6 +20,8 @@
 
       @media (min-width: 1171px) {
         right: 235px;
+        position: absolute;
+        top: -58px;
```

Comment:
We need to think about what are we going to do with this. We can’t keep adding tech debt forever, eventually we’ll need to clean up the code.

Reply:
This selector has been a pain from the beginning, I agree with you but we have other priorities to focus on for now what adds more value.